python/PiFinder/main.py

```diff
@@ -709,7 +710,7 @@ def main(script_name=None):
         from PiFinder import keyboard_server as keyboard
 
     if args.log:
-        datenow = datetime.now()
+        datenow = datetime.datetime.now()
         filehandler = f"PiFinder-{datenow:%Y%m%d-%H_%M_%S}.log"
```

Comment:
I know this line has been here for a bit, but in the field the PiFinder likely won't have the correct date time.  In other areas where a distinct filename is needed, the UID of the session is used in place of the date time stamp.

It makes the files a bit harder to ultimately use, as they are just random strings, but at least all the files generated from the same execution of the software have the same filename prefix.  

After merging, I'll adjust this line to use the session UID unless you have some other idea on how to deal with the lack of solid date and timekeeping in the field without internet?

Reply:
isn't the time coming from the GPS?

Reply:
It could... although I've not setup GPSd to set the system time.  Even if it was setting the system time, it's very unlikely there would be a fix by the time this code runs during the first minute or so of system startup.  It is kind of a bummer not to have reliable date/time from system boot... I've definitely been spoiled by the era of network connected everything!